### PR TITLE
feat(data-explorer): i18n for data explorer 2

### PR DESCRIPTION
**Pull request by connoratrug**
- Replace fixed labels with i18n labels

Related to: https://github.com/molgenis/molgenis/pull/9160
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Conventional commits (squash if needed)
- [ ] No warnings during install
- [ ] Updated javascript typing

### CODE REVIEW

**Comment by connoratrug** [comment]
[#7326 As explorer user i want localization [1] (custom labels)](https://trello.com/c/urMEjTiW/389-7326-as-explorer-user-i-want-localization-1-custom-labels)

**Review by jvanveen — COMMENTED** [review]
Nog een foutje in de CI; iets met $t

File: packages/data-explorer/tests/unit/views/ToolbarView.spec.ts [hunk 1]
```diff
@@ -20,7 +22,8 @@ describe('ToolbarView.vue', () => {
     $route: { params: {}, query: {} },
     $router: {
       push: jest.fn()
-    }
+    },
+    $t
```
Comment (jvanveen) [inline packages/data-explorer/tests/unit/views/ToolbarView.spec.ts:26]: Hoort die hier?

**Review by jvanveen — APPROVED** [review]

File: packages/data-explorer/tests/unit/views/ToolbarView.spec.ts (hunk 1 above)
Comment (connoratrug) [reply packages/data-explorer/tests/unit/views/ToolbarView.spec.ts:26]: yes

**Pull request merged**